models/daily_spend.sql

```diff
@@ -143,14 +148,42 @@ cloud_services_spend_daily as (
     group by 1, 2, 3, 4
 ),
 
+
+credits_billed_daily as (
+    select
+        convert_timezone('UTC', stg_metering_history.start_time)::date as date,
+        sum(credits_used_compute) as daily_credits_used_compute,
+        sum(credits_used_cloud_services) as daily_credits_used_cloud_services,
+        greatest(daily_credits_used_cloud_services - daily_credits_used_compute * 0.1, 0) as daily_billable_cloud_services
+    from {{ ref('stg_metering_history') }}
+    group by 1
+),
+
+cloud_services_spend_daily as (
+    select
+        _cloud_services_spend_daily.date,
+        _cloud_services_spend_daily.service,
+        _cloud_services_spend_daily.storage_type,
+        _cloud_services_spend_daily.warehouse_name,
+        _cloud_services_spend_daily.database_name,
+        _cloud_services_spend_daily.credits_used_cloud_services*_cloud_services_spend_daily.effective_rate as spend,
+
+        (div0(_cloud_services_spend_daily.credits_used_cloud_services, credits_billed_daily.daily_credits_used_cloud_services) * credits_billed_daily.daily_billable_cloud_services) * _cloud_services_spend_daily.effective_rate as spend_net_cloud_services
+    from _cloud_services_spend_daily
+    inner join credits_billed_daily on
+         _cloud_services_spend_daily.date=credits_billed_daily.date
+
+)
```

Comment:
this uses the same logic as `cost_per_query` model:

1. find out how many cloud services credits should be billed each day
2. for each warehouse, find out the cloud services fraction of total cloud services credits for that day
3. multiply that proportion by the cloud services credits that should be billed that day